FreewheelSampleApp/app/src/main/java/com/ooyala/sample/players/CustomConfiguredFreewheelPlayerActivity.java

```diff
@@ -102,6 +104,19 @@ public void update(Observable arg0, Object arg1) {
     if (arg1 == OoyalaPlayer.TIME_CHANGED_NOTIFICATION) {
       return;
     }
+
+
+
+    // Hook to write Notifications to a temporary file on the device/emulator
+    // Keeps track of incoming notifications and makes sure count is right
+    count++ ;
+    String text="Notification Received: " + arg1 + " - state: " + player.getState() + "count: " +count;
```

Comment:
This string is basically the same as the one on line 116 it seems? How about making that a variable, and then appending the "count:"+count part when writing to the sdcard log? Just seems like a nice opportunity to tighten/tidy those strings up together.